charm-slurmctld/src/omnietcd3.py

```diff
@@ -0,0 +1,74 @@
+"""Omnivector wrapper for etcd3gw."""
+# heavily copied from Calico project
+
+import logging
+
+from etcd3gw.client import Etcd3Client
+from etcd3gw.exceptions import Etcd3Exception
+
+logger = logging.getLogger(__name__)
+
+
+class Etcd3AuthClient(Etcd3Client):
+    """Handle etcd3 requests with auth."""
+    def __init__(self, host='localhost', port=2379, protocol="http",
+                 ca_cert=None, cert_key=None, cert_cert=None, timeout=None,
+                 username=None, password=None, api_path="/v3/"):
+        """Initialize class."""
+        super(Etcd3AuthClient, self).__init__(host=host,
```

Comment:
for python 3, the args to super can be implicit:
```suggestion
        super().__init__(...)
```

Reply:
Oh, nice to know! But I do prefer to follow PEP20 and make that explicit.